man/journalctl.xml

```diff
@@ -407,6 +407,12 @@
             precision.</para></listitem>
           </varlistentry>
 
+          <varlistentry>
+            <term><option>--truncate-newline</option></term>
+
+            <listitem><para>Truncate each message so that only the first line is displayed.</para></listitem>
```

Comment:
maybe say "Truncate each log message at the first newline character on output, so that only the first line of each message is displayed."